README.md

```diff
@@ -11,60 +11,70 @@ on IBM Blockchain.  The components of the demo are:
+
+- [CP-Web - Branch v2.0](https://github.com/ibm-blockchain/cp-web/tree/v2.0)
+	- Works with Hyperledger fabric `v0.6-developer-preview`
+
+If for some reason you need instructions for the v0.4.2+ level of the service, they are here, but support of these instructions is
+best effort only.
+
+- [CP-Web - Branch v1.0](https://github.com/ibm-blockchain/cp-web/tree/v1.0) (Deprecated)
+	- Works with Hyperledger fabric `v0.5-developer-preview`
+	- IBM Bluemix Blockchain Service `v0.4.2+`
+
 ## Getting Started
 
-1. Deploy the demo to your [IBM Bluemix](https://www.bluemix.net/) account using the button above.
+1. Deploy the demo to your [IBM Bluemix](https://www.bluemix.net/) account using this button [![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM-Blockchain/cp-web.git)  
 
-##### OR
+`OR`  
 
-1. Clone this repository.
-2. Create an instance of the IBM Blockchain service in the Bluemix catalog.
-3. Copy the credentials from the service into the file 'my_creds.json'.
-4. Make sure the key/value store only has values for your current network (See below).
-5. Run these commands in the cloned directory:
+1. Deploy the app on my local machine, connecting to an IBM Blockchain network running in Bluemix  - [instructions](#manbluenetwork)
 
-```shell
-npm install
-gulp
-```
 
-These credentials can be obtained from the "Service Credentials" tab of the Bluemix service. They are
-in the form:
-
-```json
-{
-  "credentials": {
-    "peers": [
-      {
-        "discovery_host": "169.53.62.121",
-        "discovery_port": "40275",
-        "api_host": "169.53.62.121",
-        "api_port": "40276",
-        "type": "peer",
-        "network_id": "4b21f2f9-4d10-4946-a0df-f91ac09dbc03",
-        "id": "4b21f2f9-4d10-4946-a0df-f91ac09dbc03_vp1",
-        "api_url": "http://169.53.62.121:40276"
-      }
-    ],
-    "users": [
-      {
-        "username": "user_type0_b7c7a1e545",
-        "secret": "89ce33e4e6"
-      }
-    ]
-  }
-}
-```
+# <a name="manbluenetwork"></a>Deploy the CP-Web App locally and connect to an IBM Blockchain Network running in Bluemix:
+
+1.  Follow these instructions to [Set up your environment for running the demos](https://github.com/ptippett/marbles/blob/break_out_common_sections/demo_prereqs)
+
+1.  Clone the CP-Web app to your local system so you can run it here
+To do this, run ```git clone http://gopkg.in/ibm-blockchain/cp-web.v2``` to clone the v2.0 branch to your local system.  
+
+1.  Follow the instructions to [Set up a new bluemix network or grab credentials from an existing network](https://github.com/ptippett/marbles/blob/break_out_common_sections/create_blockchain_bluemix.md)
```

Comment:
We should link to the bluemix docs for this:
https://console.ng.bluemix.net/docs/services/blockchain/index.html?pos=2